.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Build and Test
+on: [ pull_request ]
+
+jobs:
+  build_and_test:
+    runs-on: ubuntu-latest
+    name: Build and Test
+    steps:
+    - name: Checkout this commit
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: C++ Style Check (compiled)
+      uses: Dal-Papa/actions-uncrustify@v1.2
+
+    - run: sudo apt-get update
+    - run: sudo apt-get install software-properties-common
+    - run: sudo apt-get update
+    - run: sudo apt-get install scons g++-9 libboost-dev libboost-program-options-dev libboost-regex-dev libboost-filesystem-dev libmariadbclient-dev libmariadb3 ccache
+    - run: scons -C code -j4 -Q ccache=1 pretty=0 sanitize=1 shared=0 check sneezy
```

Comment:
It'll hang, as Sneezy won't exit. Besides, you don't have the DB set up.
Would it be possible instead to use a compose file, combining DB with a MUD client logging in as admin and shutting down the MUD? Maybe also run a couple of commands. We could use this to build up a suite of blackbox tests.

Reply:
oops, it'll just compile, not run it. Disregard the bit about hanging.